uat/src/main/java/com.aws.greengrass/FileSteps.java

```diff
@@ -127,19 +155,9 @@ public void verifyActiveFile(String componentName) {
         if (!platform.files().exists(logsDirectory)) {
             throw new IllegalStateException("No logs directory");
         }
-
-        List<File> componentFiles = Arrays.stream(logsDirectory.toFile().listFiles())
-                .filter(File::isFile)
-                .filter(file -> file.getName().startsWith(componentName))
-                .sorted(Comparator.comparingLong(File::lastModified))
-                .collect(Collectors.toList());
-
-        assertEquals(1, componentFiles.size());
```

Comment:
If you remove this assertion we are no longer accomplishing `verify the rotated files are deleted`. Add it back